<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>932</width>
    <height>639</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>741</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>741</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_pos">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>540</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>R=0         G=0         B=0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>170</y>
      <width>120</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>RGB channel</string>
    </property>
    <widget class="QCheckBox" name="checkBox_R">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>35</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>R</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_G">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>35</width>
       <height>20</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>G</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_B">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>35</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>B</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>411</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-100</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="sliderValueLBL">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>540</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>932</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionZoomInOut">
   <property name="text">
    <string>ZoomInOut</string>
   </property>
   <property name="toolTip">
    <string>Zoom </string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
